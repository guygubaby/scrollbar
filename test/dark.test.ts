import { defaultOptions, generateCss } from '@bryce-loskie/scrollbar'
import { expect, test } from 'vitest'

test('dark mode, should generate dark mode css', () => {
  expect(true).toBe(true)

  const css = generateCss({
    ...defaultOptions,
    darkMode: 'class',
    darkOptions: {
      thumbColor: 'red',
    },
  })

  expect(css).toMatchSnapshot()
})
