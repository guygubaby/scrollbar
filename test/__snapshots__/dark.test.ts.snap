// Vitest Snapshot v1

exports[`dark mode, should generate dark mode css 1`] = `
"
  :root {
    --scrollbar-width: 8px;
    --scrollbar-height: 8px;
    --scrollbar-track-radius: 4px;
    --scrollbar-thumb-radius: 4px;
    --scrollbar-track-color: #f5f5f5;
    --scrollbar-thumb-color: #ccc;
    --scrollbar-thumb-hover-color: #999;
    scrollbar-color: var(--scrollbar-thumb-color) var(--scrollbar-track-color);
    scrollbar-width: thin;
  }
  "
`;
